docs/notebooks/explicit_constraints.pct.py

```diff
@@ -125,26 +105,23 @@ def constraint(input_data):
 
 
 # %% [markdown]
-# ## Acquisition function
+# ## Constrained optimization method
 #
-# We can construct the _expected constrained improvement_ acquisition function similar to the [inequality-constraints notebook](inequality_constraints.ipynb). However, instead of using probability of feasibility with respect to the constraint model, we construct feasibility from the explicit input constraints. Feasibility is calculated by passing all the constraints residuals (to their respective limits) through a smoothing function and taking the product.
+# ### Acquisition function (constrained optimization)
 #
-# Note this method penalises the expected improvement acquisition outside the feasible region. The optimizer uses unconstrained L-BFGS method to find the max of the acquistion function.
+# We can construct the _expected improvement_ acquisition function as usual. In order to handle the constraints, the search space must be passed as a constructor argument.
```

Comment:
I don't understand "the search space must be passed as a constructor argument."

Also, I think it would make sense to explicitly build the search space in the notebook. This is possibly the main thing a user would have to replicate on their own problem?

Reply:
perfect